src/devices/Is31fl3731/Is31fl3731.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents an IS31FL3731 LED Matrix driver
+    /// </summary>
+    // Datasheet: https://cdn-learn.adafruit.com/assets/assets/000/030/994/original/31FL3731.pdf
+    // Product: https://www.adafruit.com/product/2946
+    // Product: https://www.adafruit.com/product/2974
+    public abstract class Is31fl3731
+    {
+        // Register addresses
+        // Command register
+        // Points pages one to nine
+        // table 2 in datasheet
+        private const byte COMMAND_REGISTER = 0xFD;
```

Comment:
rather than private const, a nice enum is always preferable especially in that case where you have quite a lot of them.

Reply:
I took your advice.

I first tried enums. That looked pretty rough to me since I had to add a cast at every callsite: https://github.com/dotnet/iot/pull/1926/commits/f3c7b6334af0b3547c035b0c82ab2f7fe5d4cd44

I then switched to static classes. That resolved the issue: https://github.com/dotnet/iot/pull/1926/commits/ecd652f87f44bd0de2671b8420d41fdda75a6e8f

I left the API signatures as `byte`. The reason is that there are multiple registers and their values overlap. Also, one of the registers takes values 0-7 for 8 pages of data. Forcing the use of an enum or static class to iterate from `0..7` doesn't make much sense. 

Seem better?